dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -359,5 +343,12 @@ WORKDIR /azp
 COPY ./start.sh .
 RUN chmod +x start.sh
 
+# Install python3 as default python
+RUN ln -sf /usr/bin/python3 /usr/bin/python \
+    && ln -sf `which pip3` /usr/bin/pip
+
+COPY 0001-Fix-getattr-AttributeError-in-multi-thread-scenario.patch /tmp/
```

Comment:
Is it possible to combine this COPY with `COPY ./start.sh .`? Also combine the last `RUN` instructions in one line? 
I have this ask is because combine them together can reduce some layers in the docker image.

Reply:
Please see latest commit.